### PR TITLE
Vanity Domain CNAME Values

### PR DESCRIPTION
**Pull request by rachelwhitton**
Re: BUGS-1473

## Effect
PR includes the following changes:
- Document DNS requirements to use platform domains as recommended on sites with vanity domains enabled

@nstielau @ari-gold @lauren-kelly

### CODE REVIEW

File: source/_docs/vanity-domains.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ At your DNS provider, create a wildcard CNAME record pointing to our edge. If yo
 
  - **Environment URLs are permanent.** If an organization is removed as the supporting organization, any environment created during its association will keep the original URL after removal. Paid sites can add custom domains to any environment, as a workaround for those wishing to use different URL's after launch and disassociation of the site with the organization.
 
+ - To ensure maximum uptime, configure DNS of Custom domains (e.g., `www.example.com` on Live) using the environment's Platform domain for the CNAME record value as recommended within the **<span class="glyphicons glyphicons-global"></span> Domains / HTTPS** page, never a Vanity domain:
```
Comment (alexfornuto) [inline source/_docs/vanity-domains.md]: This may be one of the most densely packed sentences I've ever come across, and I'll need help deciphering it.

Comment (rachelwhitton) [inline source/_docs/vanity-domains.md]: Halp.

**Comment by lauren-kelly** [comment]
++++++1

**Comment by rachelwhitton** [comment]
@ari-gold Whoops! Sorry I merged this before I saw your response. Do you know how vanity domains are configured if not by the existing wildcard instructions?

**Comment by alexfornuto** [comment]
@rachelwhitton @ari-gold do we need a new issue to process any remaining work to be done? I'm processing and archiving this card at the moment.

**Comment by ari-gold** [comment]
Think we're good, for now.

**Pull request merged**